plugins/org.jboss.tools.livereload.ui/plugin.xml

```diff
@@ -82,9 +82,9 @@
                  <iterate
                        ifEmpty="false"
                        operator="or">
-                    	<adapt
-							type="org.eclipse.wst.server.ui.IServerModule">
-						</adapt>
+                    	<instanceof
+							value="org.eclipse.wst.server.ui.IServerModule">
+						</instanceof>
```

Comment:
So actually this changes breaks openshift support. The menu is enabled at the openshift server level, which is not an instance of IServerModule, but adapts to IServerModule. It was broken in openInExternalDeviceWebBrowserViaQRCode, which was fixed in 67d2360